zulip_bots/zulip_bots/bots/mention/test_mention.py

```diff
@@ -30,3 +30,11 @@ def test_get_alert_id(self) -> None:
 
         with self.mock_http_conversation('get_alert_id'):
             self.assertEqual(bot_test_instance.get_alert_id('TEST'), 'TEST')
+
+    def test_get_mentions(self) -> None:
+        bot_test_instance = MentionHandler()
+        bot_test_instance.access_token = 'TEST'
+        bot_test_instance.account_id = 'TEST'
+
+        with self.mock_http_conversation('get_mentions'):
+            self.assertEqual(bot_test_instance.get_mentions('TEST'), [])
```

Comment:
Can we test this with more data than an empty list?

Reply:
I tried. It was difficult getting demo data with a free account.

Reply:
What I would do in this case is add mentions with text like "simulated mention for test purposes #1" and "simulated mention for test purposes #2", just so we can exercise the code a bit.  (The test still won't really ensure that we are correct for the actual API, but it will at least ensure that the code works as intended.)

Reply:
Ok. Will do that today itself.